IPython/core/excolors.py

```diff
@@ -155,6 +156,12 @@ def exception_colors():
         Normal = C.Normal,
         ))
 
+    # Hack: the 'neutral' colours are not very visible on a dark background on
+    # Windows. Since Windows command prompts have a dark background by default, and
+    # relatively few users are likely to alter that, we will use the 'Linux' colours,
+    # designed for a dark background, as the default on Windows.
+    if os.name == "nt":
+        ex_colors.add_scheme(ex_colors['Linux'].copy('Neutral'))
```

Comment:
Something like this https://github.com/segevfiner/ipython/commit/50c3fc12a6f8ebe55e2ea848e615974f582b5b28?

I guess @Carreau will decide?